bitbots_vision_tools/scripts/extract_from_rosbag.py

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+import os
+import cv2
+import rosbag
+import argparse
+import numpy as np
+from cv_bridge import CvBridge
+from sensor_msgs.msg import Image
+
+try:
+    input = raw_input
+except NameError:
+    pass
+
+def yes_or_no_input(question, default=None):
+    # type: (str) -> bool
+    """
+    Prints a yes or no question and returns the answer.
+    
+    :param str question: Yes or no question
+    :param bool default: Default answer, if empty answer
+    :return bool: Input answer
+    """
+    answer = None
+    reply = None
+    extention = None
+
+    if default is None:
+        extention = " [y|n]"
+    elif default == True:
+        extention = " [Y|n]"
+    elif default == False:
+        extention = " [y|N]"
+
+    while answer is None:
+        reply = str(input(question + extention + ": ")).lower().strip()
+        if default is not None and reply == "":
+            answer = default
+        elif reply[:1] == 'y':
+            answer = True
+        elif reply[:1] == 'n':
+            answer =  False
+    return answer
+
+def int_input(question, min_int=None, max_int=None):
+    # type: (str, int, int) -> int
+    """
+    Prints a question about a int value and returns the answer.
+    
+    :param str question: Int question
+    :param int min_int: Minimum input value to be accepted
+    :param int max_int: Maximum input value to be accepted
+    :return int: Input answer
+    """
+    answer = None
+    reply = None
+    extention = None
+
+    # Construct full question with min and max
+    if min_int is not None and max_int is None:
+        extention = " [MIN: {}]".format(min_int)
+    elif min_int is None and max_int is not None:
+        extention = " [MAX: {}]".format(max_int)
+    elif min_int is not None and max_int is not None:
+        if not min_int <= max_int:
+            raise ValueError("min_int must be smaller or equal to max_int.")
+        else:
+            extention = " [{} - {}]: ".format(min_int, max_int)
+
+    while answer is None:
+        try:
+            reply = int(input(question + extention + ": ").strip())
+        except ValueError:
+            pass
+
+        # Check for min and max conditions
+        if reply is not None:
+            if min_int is None and max_int is None:
+                answer = reply
+            elif min_int is not None and max_int is None and min_int <= reply:
+                answer = reply
+            elif min_int is None and max_int is not None and reply >= max_int:
+                answer = reply
+            elif min_int and max_int and min_int <= reply <= max_int:
```

Comment:
0 evaluates to False. Therefore `is not None` must be used or the script breaks on `min_int = 0`.